### PR TITLE
Add Aqara integration

### PR DESCRIPTION
**Pull request by chenliuyang1989**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/71534
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/3411
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by chenliuyang1989** [comment]
> It seems that this PR is targeted against an incorrect branch. Documentation updates which apply to our current stable release should target the `current` branch. Please change the target branch of this PR to `current` and rebase if needed. If this is documentation for a new feature, please add a link to that PR in your description. (message by DocsTargetBranch)

this is the PR link:
https://github.com/home-assistant/core/pull/71534

File: source/_integrations/aqara.markdown [hunk 1]
```diff
@@ -0,0 +1,57 @@
+---
+title: Aqara
+description: Instructions on how to set up the aqara devices within Home Assistant..
```
Comment (Daneish) [inline source/_integrations/aqara.markdown:3]:
```suggestion
description: Instructions on how to set up the Aqara integration within Home Assistant.
```

File: source/_integrations/aqara.markdown [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+ha_codeowners:
+  - '@wumao'
+  - '@chenliuyang1989'
+ha_domain: aqara
+ha_platforms:
+  - binary_sensor
+ha_integration_type: integration
+---
+
+Aqara integration allows users to integrate binary devices and scenes on the Aqara Home app into Home Assistant. After the account authorization, the supported devices and scenes will be automatically synchronized to the Home Assistant.
```
Comment (Daneish) [inline source/_integrations/aqara.markdown:18]:
```suggestion
The Aqara integration allows users to integrate binary devices and scenes from the Aqara Home app into Home Assistant. After the account authorization, the supported devices and scenes will be automatically synchronized with Home Assistant.
```

File: source/_integrations/aqara.markdown [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+## Configuration of the Aqara IoT Platform
+
+### PREREQUISITES
+
+- You will need to create an account in the Aqara Home app.
+- Your devices also need first to be added in the Aqara Home app.
```
Comment (Daneish) [inline source/_integrations/aqara.markdown:25]:
```suggestion
- Your devices also need to be added in the Aqara Home app.
```

File: source/_integrations/aqara.markdown [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+### ADD DEVICES BY AQARA HOME APP
```
Comment (Daneish) [inline source/_integrations/aqara.markdown:27]: The 'Add Devices to App' section may not be required as it isn't included on other integrations such as Xiaomi Miio or Tuya. If you remove it, you can remove the this:
````
## Configuration of the Aqara IoT Platform

### PREREQUISITES 
````

And replace it with:
````
## Prerequisites
````

File: source/_integrations/aqara.markdown [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+1. In the left navigation bar of Home Assistant, click on **Configuration**.
+3. Select **Devices and Services** from the menu.
+4. In the bottom right corner, click on the **ADD INTEGRATION** button.
+5. Search and select **Aqara**.
+6. Complete the configuration according to the page introduction.
+
+**Country**
+
+Choose the country as Aqara Home app. Currently support China mainland, Europe, South Korea, Russia, United States, Singapore.
```
Comment (Daneish) [inline source/_integrations/aqara.markdown:49]:
```suggestion
Choose the country used in the Aqara Home app. Currently supports China mainland, Europe, South Korea, Russia, United States, and Singapore.
```

**Comment by Daneish** [comment]
Thanks very much for the new integration. I've made a couple of suggestions above to tidy up the documentation. Please just take these as suggestions (not mandatory), as I am new to the documentation review process.

File: source/_integrations/aqara.markdown [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+
+**Account**
+
+Aqara Home app account.
+
+**Password**
+
+The password of your Aqara Home app account.
```
Comment (Daneish) [inline source/_integrations/aqara.markdown:57]:
```suggestion
Enter the Aqara Home app account password.
```

File: source/_integrations/aqara.markdown [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+
+1. Download and login Aqara Home app.
+
+2. Click on the **Add Device** button on the Device page.
+
+3. Select device type and complete the configuration according to the introduction. 
+
+   > Note: Add the Hub first, then add the sub-devices.
+
+## Configuration
```
Comment (Daneish) [inline source/_integrations/aqara.markdown:37]: The Configuration heading and steps 1-6 could possibly be replaced with:
````
{% include integrations/config_flow.md %}
````
But I will leave that for another reviewer to confirm.

File: source/_integrations/aqara.markdown (hunk 6 above)
Comment (Daneish) [inline source/_integrations/aqara.markdown:57]:
```suggestion
{% configuration_basic %}
Country:
  description: The country used in the Aqara Home app (currently supports China mainland, Europe, South Korea, Russia, United States, and Singapore).
Account:
  description: The Aqara Home app account name.
Password:
  description: The Aqara Home app account password.
{% endconfiguration_basic %}
```

**Comment by frenck** [comment]
Closing PR as the parent PR has been closed.

**Pull request closed without merging**